infrastructure/validators/operational-tasks/how-to-upgrade-validator.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Upgrade a Validator Node
+description: Learn how to upgrade your Polkadot validator node, manage session keys, and perform seamless server maintenance without disrupting network validation.
+---
+
+# Upgrade a Validator Node
+
+## Introduction
+
+Validators perform critical functions for the network by [backing and including blocks](https://wiki.polkadot.network/docs/learn-parachains-protocol){target=\_blank}. Validators may have to go offline for short-periods of time to upgrade client software or to upgrade the host machine. Usually, standard client upgrades will only require you to stop the service, replace the binary and
+restart the service. This operation can be executed within a session.
+
+Validators may also need to perform long-lead maintenance tasks that will span more than one
+session. Under these circumstances, an active validator may chill their stash and be removed from
+the active validator set. Alternatively, the validator may substitute the active validator server
+with another allowing the former to undergo maintenance activities.
+
+This guide will provide an option for validators to seamlessly substitute an active validator server
+to allow for maintenance operations. The process can take several hours, so make sure you understand the instructions first and plan accordingly.
+
+!!!tip "Keep an eye out on new releases from the community"
+    Upgrade or downgrade accordingly.
+
+## Prerequisites
+
+Before continuining with this guide, ensure you have:
```

Comment:
Please add some things the user should already be familiar with, might want to read first, stuff to ensure they have installed, etc., before continuing with this guide.